changelog.yaml

```diff
@@ -14,6 +14,21 @@
 
 #      jiraIssueNumber: <Internal Jira issue number>
 
+- version: "NEXT"
+  date: TBD
+  changes:
+  - type: enhancement
+    impact: minor
+    title: Make elasticSearch IndexURL configurable in PipelineRun custom resource objects.
+    description: >-
+      There exists a default elasticSearch IndexURL per cluster. It reduces flexibility to
+      push logs to different indeces in elasticSearch. This change allows developers to
+      make elasticSearch IndexURL configurable in Steward-PipelineRun object as a parameter.
+    warning: ""
+    upgradeNotes: ""
+    pullRequestNumber: 172
+    jiraIssueNumber: 984
+
 - version: "NEXT"
   date: TBD
   changes:

```

Comment:
```suggestion
```
You should not have two entries with version NEXT but two changes in version NEXT.